datasets/libsvm.py

```diff
@@ -1,40 +1,27 @@
-from benchopt import BaseDataset
-
-from benchopt import safe_import_context
-
+from benchopt import BaseDataset, safe_import_context
 
 with safe_import_context() as import_ctx:
     from libsvmdata import fetch_libsvm
-    from sklearn.preprocessing import StandardScaler
 
 
 class Dataset(BaseDataset):
 
     name = "libsvm"
 
     parameters = {
-        'dataset': ["bodyfat", "leukemia", "rcv1.binary", "YearPredictionMSD"],
+        "dataset": ["bodyfat", "finance", "finance-tf-idf", "YearPredictionMSD"],
     }
 
-    install_cmd = 'conda'
-    requirements = ['pip:git+https://github.com/mathurinm/libsvmdata@main']
-    references = [
-        "C. Chang and CJ. Lin, "
-        "'ACM transactions on intelligent systems and technology (TIST)', "
-        "Acm New York, USA vol 2 (2011)."
-    ]
+    install_cmd = "conda"
+    requirements = ["pip:libsvmdata"]
```

Comment:
are you sure this does not break ? some datasets may be in libsvmdata dev that are not published to PyPI yet